src/iop/demosaic.c

```diff
@@ -408,21 +408,33 @@ void modify_roi_in(
   roi_in->scale = 1.0f;
 
   dt_iop_demosaic_data_t *data = (dt_iop_demosaic_data_t *)piece->data;
-  const int method = data->demosaicing_method;
+  const dt_iop_demosaic_method_t method = data->demosaicing_method;
   const gboolean passthrough = (method == DT_IOP_DEMOSAIC_PASSTHROUGH_MONOCHROME) ||
-                               (method == DT_IOP_DEMOSAIC_PASSTHR_MONOX);
-
+                               (method == DT_IOP_DEMOSAIC_PASSTHR_MONOX) ||
+                               (method == DT_IOP_DEMOSAIC_PASSTHROUGH_COLOR) ||
+                               (method == DT_IOP_DEMOSAIC_PASSTHR_COLORX);
   // set position to closest sensor pattern snap
```

Comment:
Worth changing this comment to reflect that alignment only moves origin up/left, not to closest.

Reply:
Right. missed it :-)